### PR TITLE
No user or keys

### PR DESCRIPTION
**Pull request by JaredGordon**

### CODE REVIEW

File: modules/ops_manager/iam.tf [hunk 1]
```diff
@@ -1,87 +1,7 @@
...
-resource "aws_iam_policy" "ops_manager_role" {
-  name   = "${var.env_name}_ops_manager_role"
-  policy = "${data.aws_iam_policy_document.ops_manager.json}"
-}
-
-resource "aws_iam_role_policy_attachment" "ops_manager_policy" {
-  role       = "${aws_iam_role.ops_manager.name}"
-  policy_arn = "${aws_iam_policy.ops_manager_role.arn}"
+data "aws_iam_role" "ops_manager" {
+  name = "DIRECTOR"
```
Comment (voor) [inline modules/ops_manager/iam.tf]: This seems like it's unintentional in this PR, are you hard-coded the role to a pre-existing out-of-band created policy?

Comment (JaredGordon) [reply modules/ops_manager/iam.tf]: throw this PR out: we were merging to our own forked "master" and must have pushed the "create PR" button on the upstream instead. Sorry about that.

Comment (JaredGordon) [reply modules/ops_manager/iam.tf]: closing the PR!

**Comment by JaredGordon** [comment]
delete this PR, created in error.

**Comment by JaredGordon** [comment]
oops. deleting

**Comment by JaredGordon** [comment]
Yes, sorry about this, we were merging into our own, forked master and must
have hit the button with the upstream master instead. Please ignore. I've
closed the PR.

**Comment by voor** [comment]
For what it's worth, I think it might be worth looking into creating a PR to just remove the access key for Ops Manager, instance IAM are much more secure and we should be secure by default. @zachgersh

**Pull request closed without merging**